build.gradle.kts

```diff
@@ -19,10 +19,14 @@ plugins {
     signing
 }
 
+java {
+  sourceCompatibility = JavaVersion.VERSION_11
+  targetCompatibility = JavaVersion.VERSION_11
+}
+
 repositories {
     mavenCentral()
     maven("https://maven.scijava.org/content/groups/public")
-    maven("https://jitpack.io")
```

Comment:
I guess we want to get rid of jitpack because of reproducibility issues and simplifying the maven repo usage? That is fine with me.

I would just share that this is the simplest way to get some of the N5 and BDV artifacts (n5-viewer comes to mind) in a reliable semi-reproducible way, and I am using those artifacts for the [OpenOrganelle demo](https://github.com/scenerygraphics/sciview/tree/openorganelle-demo).

Reply:
N5 and BDV artifacts are available at release versions from maven.scijava.org. And jitpack is also proxied by the maven.scijava.org public group. I see the value in adding the jitpack repo separately in local dev scenarios. But do we need it for the officially pushed build system?

Reply:
Yeah, I am aware that they're on scijava maven, but N5 artifact releases are slow compared to the development. There also has been some API change in N5/BDV that requires me to use SNAPSHOTs for doing the advanced stuff that will be required for the sciview paper demo. 

I'm still very fine with getting jitpack out of the official build. I'm sure we can nudge folks to get things out of SNAPSHOT stage. I personally was very strongly attached to keeping jitpack in the list because I've often had short amounts of time to work on things, and I ended up losing most of that time just refreshing my dev environment to either locally link projects or do local SNAPSHOT builds. I am convinced that just pointing to a jitpack hash and having it work has saved me hundreds of hours.

Merging.